### ISSUE TITLE
Clash between max-len and prefer-template

### ISSUE THREAD

**Issue by callumlocke** [airbnb/javascript#814]
This errors because of [max-len](http://eslint.org/docs/rules/max-len):

```js
    let report = `<span style="color:red;font-weight:bold;font:bold 20px sans-serif">${headline}</span>`
```

...but if I try to break it up over multiple lines by separating it into strings and concatenating with `+`, it errors because of [prefer-template](http://eslint.org/docs/rules/prefer-template).

What am I supposed to do in this situation?

**Comment by ljharb** [comment]
Temporarily disable `max-len`. I've filed https://github.com/eslint/eslint/issues/5805 to add an `ignoreStrings` option to the rule which would make this example pass.

**Comment by ljharb** [comment]
This is pending https://github.com/eslint/eslint/pull/7049 to be merged and deployed.

**Comment by ljharb** [comment]
This is now resolved in v8.0.0 of `eslint-config-airbnb-base`.
